src/main/webapp/app/entities/quiz/quiz-group.model.ts

```diff
@@ -0,0 +1,6 @@
+import { BaseEntity } from 'app/shared/model/base-entity';
+
+export class QuizGroup implements BaseEntity {
+    public id?: number;
```

Comment:
```suggestion
```
`id` is defined in BaseEntity or do you explicitly require the `public` modifier? To my understanding, this should make no difference as `id` is public per default.